bundles/model/org.openhab.model.script/src/org/openhab/model/script/internal/actions/TimerImpl.java

```diff
@@ -30,6 +33,10 @@
  * @author Kai Kreuzer
  * @since 1.0.0
  *
+ * @author Peter Broucke
+ * @since 1.0.1
+ * Extended by Peter Broucke to support CronSchedulers
```

Comment:
please enhance the general class description, add yourself as author directly beneath Kai and leave @since unchanged (it shall indicate when the class has been introduced.